server/test/rating.spec.js

```diff
@@ -1,7 +1,8 @@
 import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../../index';
-import userFaker from './helpers/userFakeData';
+import users from '../utils/users';
+// import userFaker from './helpers/userFakeData';
```

Comment:
Please remove this line of code you commented out, I believe it is not doing anything, right?

Reply:
done..